extensions/amp-story/amp-story-analytics.md

```diff
@@ -64,6 +64,10 @@ The `story-bookend-enter` trigger is fired when the bookend is shown to the user
 
 The `story-bookend-exit` trigger is fired when the bookend is dismissed by the user.
 
+### Bookend click trigger (`"on": "story-bookend-click"`)
+
+The `story-bookend-click` trigger is fired when a user clicks a link inside the bookend.
```

Comment:
Should we mention that it is triggered with the extra variables listed below?

Reply:
Do these need to be explicitly stated int he analytics configuration to be tracked, or are all variables triggered under all circumstances?

Reply:
Yes, you need to declare them in your configuration if you want to track the variables. 

You could only track the trigger without the variables, too. This would be useful if you are only interested in knowing the CTR in your bookend, for example.